app/main.py

```diff
@@ -1,8 +1,55 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
```

Comment:
Its better to use 1 style for opening closing brackets f.e.
(variable,
variable), or 
(
variable,
variable
)